README.md

```diff
@@ -93,19 +93,19 @@ labelled *a, b, and c*. These files (arbitrarily) define a subset of the applica
 1. In the terminal for the **private1** cluster, deploy the following:
 
    ```bash
-   kubectl apply -f ~/hipster-demo/skupper-example-grpc/deployment-ms-a.yaml
+   kubectl apply -f ~/boutique-demo/skupper-example-grpc/deployment-ms-a.yaml
```

Comment:
@nluaces following the updated README instructions, I believe that we don't need to use
`kubeapply -f ~/boutique-demo/skupper-example-grpc/...` because at line 36, the user will run
`cd boutique-demo` and so, I believe, we could consider using just: `kubeapply -f skupper-example-grpc/...`.

Reply:
Thanks! I'll change it before merging this PR.

Reply:
Done!